src/main.js

```diff
@@ -142,6 +142,7 @@ app.on("ready", async () => {
     store.get(settings.trayIcon) && addTray(mainWindow, { icon }) && refreshTray();
     store.get(settings.api) && expressModule.run(mainWindow);
     store.get(settings.enableDiscord) && discordModule.initRPC();
+    console.log(store.get(settings.mutedArtists).find((artist) => artist === "Yes"))
```

Comment:
Why is there just a new random console.log here that would end up having no use for end-users nor people that use the app in general since devtools don't work.

Reply:
that was a mistake and I meant to remove it sorry. i'll fix that